addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -426,6 +428,10 @@ QUnit.test('chat window: basic rendering', async function (assert) {
         "General",
         "should have correct thread name in header part"
     );
+    assert.ok(
+        chatWindowHeader.querySelector(`:scope .o_ChatWindowHeader_name:not(.o-clickable)`),
+        "header of channel type thread should not be clickable"
+    );
```

Comment:
use doesNotHaveClass instead.
And explain why it should not be clickable in the comment.